### PR TITLE
GAWB-3103: finish dockstore import

### PR DESCRIPTION
**Pull request by MattPutnam**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/page/external_importer.cljs
Comment (aednichols) [inline src/cljs/main/broadfcui/page/external_importer.cljs:180]: The UI uses the source repo identifier in copy, so Dockstore is accidentally lower-case:
> Importing github.com/vrockai/dockstore-whalesay:dnastack from dockstore

File: src/cljs/main/broadfcui/common/style.cljs
Comment (davidangb) [inline src/cljs/main/broadfcui/common/style.cljs:134]: heh - we have `exception-state` in two other places in the codebase. (no need to hold this PR for that)

File: src/cljs/main/broadfcui/page/external_importer.cljs
Comment (aednichols) [inline src/cljs/main/broadfcui/page/external_importer.cljs:130]: Just wanted to make sure that we are deliberately re-using the workspace's namespace as the config's namespace within the workspace?

File: src/cljs/main/broadfcui/page/external_importer.cljs
Comment (davidangb) [inline src/cljs/main/broadfcui/page/external_importer.cljs]: Since Dockstore's error messages are unhelpful, how about:

"Error loading WDL. Please verify the workflow path and version and ensure this workflow supports WDL."

(there are more error conditions than that, but I think that covers the majority)

File: src/cljs/main/broadfcui/page/external_importer.cljs
Comment (davidangb) [inline src/cljs/main/broadfcui/page/external_importer.cljs]: grammar: "cannot guarantee that the WDL and Docker image referenced by this Workflow"

File: src/cljs/main/broadfcui/page/external_importer.cljs
Comment (davidangb) [inline src/cljs/main/broadfcui/page/external_importer.cljs:180]: this will fail if the version itself contains a colon, right? Probably not worth protecting against right now, we can fix later if we realize it's a problem in practice.

**Review by davidangb — COMMENTED** [review]
minor comments inline.

File: src/cljs/main/broadfcui/page/external_importer.cljs
Comment (MattPutnam) [reply src/cljs/main/broadfcui/page/external_importer.cljs:130]: Yeah, because method config namespaces aren't really a thing any more but are still required by Agora.

File: src/cljs/main/broadfcui/page/external_importer.cljs
Comment (MattPutnam) [reply src/cljs/main/broadfcui/page/external_importer.cljs:180]: Unless dockstore supports versions with colons in them, I guess.

**Review by davidangb — COMMENTED** [review]
tentative 👍 , will review again once other PRs land and this is finalized

**Review by aednichols — APPROVED** [review]
LGTM 👍

Tested locally.

**Review by davidangb — APPROVED** [review]
:+1: for code, waiting to see tests pass

**Pull request merged**
